cli/util/uploader.go

```diff
@@ -141,82 +141,84 @@ func uploadBin(ctx *cli.Context) error {
 
 	fmt.Fprintln(ctx.App.Writer, "First block of latest incomplete batch uploaded to NeoFS container:", lastMissingBlockIndex)
 
-	if lastMissingBlockIndex > int(currentBlockHeight) {
-		fmt.Fprintf(ctx.App.Writer, "No new blocks to upload. Need to upload starting from %d, current height %d\n", lastMissingBlockIndex, currentBlockHeight)
-		return nil
-	}
+	if !ctx.Bool("skip-blocks-uploading") {
+		if lastMissingBlockIndex > int(currentBlockHeight) {
+			fmt.Fprintf(ctx.App.Writer, "No new blocks to upload. Need to upload starting from %d, current height %d\n", lastMissingBlockIndex, currentBlockHeight)
+			return nil
+		}
```

Comment:
We have `updateIndexFiles`, let's do the same for blocks: create a separate function something like `uploadBlocks`, move relevant code to this function and then we'll have:
```go
	if !ctx.Bool("skip-blocks-uploading") {
		err := uploadBlocks(...)
		if err != nil {...}
	}
	err = updateIndexFiles(...)
...

     
```

Reply:
And then `if lastMissingBlockIndex > int(currentBlockHeight) {` condition allows to skip blocks uploading in case of too low RPC height, but still continue index files uploading.

Reply:
Not implemented as suggested.